docs: delete commands for removed features

j824h opened the pull request:
The suggested commands to provide Debian alternatives to ex, view, vimdiff depend on the features removed in #2008. With the recent versions of nvim, the commands will not work as intended.

Note that the package at official Debian repository provides the [commands](https://sources.debian.org/src/neovim/0.7.2-8/debian/scripts/) by installing the executable scripts to libexec.

j824h committed:
docs: delete commands for removed features

The suggested commands to provide Debian alternatives to ex, view, vimdiff depends on the removed feature #2008.
With the recent versions of nvim, the commands will not work as intended.

Note that the package at official Debian repository provides the commands by installing the executable scripts to libexec.

clason commented:
And now you know why we removed this from the Wiki and put it under full version control ;)

dundargoc commented:
I kinda question the value of mentioning "update-alternatives" at all in `INSTALL.md`. It feels a bit akin to "if you want to alias vi to nvim, add `alias vi=nvim`. Wondering if we should remove them instead. @clason thoughts?

clason commented:
I also thought this was overly specific, compared to the other distros. But I'll defer to @jamessan.

dundargoc commented:
For reference: the ubuntu section also has an "update-alternatives" section. Naturally is looks completely different.

jamessan commented:
Yeah, I don't see a reason to mention how to use update-alternatives at all (not just for the names we no longer recognize).  There are various other ways people can mimic the old commands, and plenty of people that will have never heard of the old commands.

j824h committed:
docs: remove instructions on update-alternatives

The instructions for using update-alternatives in Debian and Ubuntu are removed.
This increases the consistency of the whole document by restricting to the scope to installation of Neovim and nvim cli commands.
If demands for the documentation specific to the platforms are raised, it would need to be suggested again.

j824h commented:
Thanks for the discussion.
So we're deciding that we restrict this document for installation of Neovim and `nvim` CLI commands.
Someone can suggest a separate issue if such contents need to be included in the documentation again.

See https://github.com/neovim/neovim/pull/27060/commits/2be85bfa64b7c9f45da463c22265902d0344a012.
